rero_ils/modules/users/jsonschemas/users/user-v0.0.1.json

```diff
@@ -37,10 +37,15 @@
       "minLength": 9,
       "default": "https://bib.rero.ch/schemas/users/user-v0.0.1.json"
     },
+    "pid": {
+      "title": "User ID",
+      "type": "number"
+    },
     "first_name": {
       "title": "First name",
       "type": "string",
       "minLength": 1,
+      "maxLength": 255,
```

Comment:
Be sure that all the common fields with invenio-userprofile has the same contraints.

Reply:
I checked with the model used by SQLAlchemy (from invenio_userprofile)